src/main/resources/public/js/recipes.js

```diff
@@ -159,8 +159,8 @@ function successOnAjaxOfRecipe(recipe) {
   $('input.inputIngInfo').keypress(function (event) {
     const idx = $(this).data('idx');
     if ($(`#quantity${idx}`).val() !== ''
-        && $(`#name${idx}`).val() !== ''
-        && event.keyCode === 13) {
+      && $(`#name${idx}`).val() !== ''
+      && event.keyCode === 13) {
```

Comment:
Uh... this should be converted to be vue-like, shouldn't it? Or maybe it is and this is stale?

Reply:
This is recipes.js, so this is our pre-vue edition.